packages/strapi-design-system/src/Field/FieldInput.js

```diff
@@ -92,6 +92,7 @@ export const FieldInput = forwardRef(({ endAction, startAction, disabled, onChan
         hasLeftAction={Boolean(startAction)}
         hasRightAction={Boolean(endAction)}
         onChange={handleChange}
+        autoComplete={disabled ? 'off' : autoComplete}
```

Comment:
I would suggest creating another PR for this autoComplete proposal, I'm not sure it has something to do with the purpose of this PR, WDYT?

Out of curiosity, what brought up the need to add this now, is it coming from a sprint task or a Github issue?

Reply:
+1 please create another PR for this proposal